Overworld.gd

```diff
@@ -28,10 +28,13 @@ func place_player():
 func _process(delta):
 	counter += delta
 	$CanvasLayer/Counter.text = "Counter: " + str(counter)
-	var cpos = $TileMap.world_to_map($Player.position)
-	$CanvasLayer/TilePos.text = str(cpos)
-	var mpos = $TileMap.world_to_map(get_global_mouse_position())
-	$CanvasLayer/MousePos.text = str(mpos)
+#	#uncomment these lines to have player pos and mouse pos appear on canvas for testing
+#	$CanvasLayer/TilePos.visible = true
+#	$CanvasLayer/MousePos.visible = true
+#	var cpos = $TileMap.world_to_map($Player.position)
+#	$CanvasLayer/TilePos.text = str(cpos)
+#	var mpos = $TileMap.world_to_map(get_global_mouse_position())
+#	$CanvasLayer/MousePos.text = str(mpos)
```

Comment:
You should create a `debug` property in Global and have this execute if it's true.

Reply:
scope creepe

Reply:
Maybe a little, but I think it's worth while.

Reply:
worth